src/index.html

```diff
@@ -5,8 +5,307 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <!-- #region header -->
+    <header class="header">
+      <a class="header__box-logo">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="log"
+           >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">Samsung</a>
```

Comment:
Make all links more meaningful.

Example: #samsung, #apple, etc.

Reply:
```suggestion
            <a href="#" class="header__link">
              Samsung
            </a>
```

Check all other similar cases.